server/routes/settings/radarr.ts

```diff
@@ -46,7 +46,13 @@ radarrRoutes.post<
       url: RadarrAPI.buildUrl(req.body, '/api/v3'),
     });
 
-    const { urlBase } = await radarr.getSystemStatus();
+    const urlBase =
+      req.body.baseUrl && req.body.baseUrl !== '/'
+        ? req.body.baseUrl
+        : await radarr
+            .getSystemStatus()
```

Comment:
Why not always fetch the base URL, and just fall back to the user-inputted value if an exception is thrown?

Also, any reason why we don't want to apply this same logic to Sonarr?